kieserver/image.yaml

```diff
@@ -142,6 +142,30 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "KIE_SERVER_KAFKA_EXT_ENABLED"
+    example: "true"
+    description: "Setting with true enable the others KIE_KAFKA_EXT environments used by Kafka, this env will be used by system prop org.kie.kafka.server.ext.disable with the opposite value"
+  - name: "KIE_SERVER_KAFKA_EXT_TOPICS"
+    example: "event=events"
```

Comment:
maybe improve this to topic_name=topic_name.
see my comments on https://github.com/jboss-container-images/jboss-kie-modules/pull/446

Reply:
ok, with message=topic_name